wpeverest/user-registration
# Addon - Customizer compatibility in customize my account

### All Submissions:






### Changes proposed in this Pull Request:
This PR provides supports for the Customizer in my account feature of User Registration Customize My Account addon.

### How to test the changes in this Pull Request:

1. Checkout to this branch after installing customize my account addon.
2. Check whether all the features in customize my account addon are working properly.

### Types of changes:




### Other information:




### Changelog entry

> Addon - Customizer compatibility in customize my account.
